FightPandemics/ViewTest.swift

```diff
@@ -0,0 +1,67 @@
+//  ProfileSignedOutView.swift
+//  FightPandemics
+//
+//  Created by newuser on 10/31/20.
+//  Copyright © 2020 Scott. All rights reserved.
+//
+import SwiftUI
+struct Success: View {
+
+
+
+    init() {
+        
+
+        UINavigationBar.appearance().titleTextAttributes = [.font: UIFont(name: "Verdana", size: 16)!]
```

Comment:
You can edit the navigation bar with modifiers in SwiftUI, not this archaic UIKit thing going on here, it's confusing seeing it mixed with SwiftUI